src/main/webapp/app/overview/teams/overview-teams.component.html

```diff
@@ -34,7 +34,7 @@ <h4 dojoTranslate="teamdojoApp.overview.teams.title"></h4>
                             <jhi-team-image [team]="teamScore.team" imageSize="small"></jhi-team-image>
                         </div>
                     <div class="team-name">
-                        <a href="#" [routerLink]="['/teams', teamScore.team.shortName]">{{teamScore.team.name}}</a>
+                        <a href="#" [routerLink]="['/teams', teamScore.team.shortName]" preserveQueryParams>{{teamScore.team.name}}</a>
```

Comment:
Produces a console warning:
> preserveQueryParams is deprecated, use queryParamsHandling instead.
```suggestion
                        <a href="#" [routerLink]="['/teams', teamScore.team.shortName]" queryParamsHandling="preserve">{{teamScore.team.name}}</a>
```